scenarioo-client/app/shared/metadata/metaDataButton.directive.ts

```diff
@@ -46,7 +46,7 @@ angular.module('scenarioo.directives').directive('scMetaDataButton', ($window, L
             initMetadataVisibleFromLocalStorage($scope, $scope.localStorageKey);
             $scope.toggleShowingMetadata = () => {
                 $scope.linkingVariable = !$scope.linkingVariable;
-                LocalStorageService.set(STEP_METADATA_VISIBLE + $scope.localStorageKey, '' + $scope.linkingVariable);
+                LocalStorageService.set(METADATA_VISIBLE_KEY + $scope.localStorageKey, '' + $scope.linkingVariable);
```

Comment:
Why is it once METADATA_VISIBLE_KEY + key and once METADATA_VISIBLE_KEY + $scope.localStorageKey?
If it should be the same, the same comment from before would apply here as well.